package com.beniaminoleone.util;

import java.sql.SQLException;

import com.beniaminoleone.model.Privileges;
import com.beniaminoleone.model.User;
import com.beniaminoleone.service.UserService;

public class Main {

	public static void main(String[] args) throws Exception {
		//User u = new User();
		//User u2 = new User();
		//User u3 = new User();
		//u.setFirstName("Beniamino");
		//u.setSurname("Leone");
		//u.setUsername("benia592");
		//u.setPassword(Encryption.encrypt("benia592"));
		//u.setPrivileges(Privileges.ADMIN);
		//u2.setFirstName("Sonia");
		//u2.setSurname("Lo Coco");
		//u2.setUsername("Sokina22");
		//u2.setPassword(Encryption.encrypt("audrey"));
		//u2.setPrivileges(Privileges.ADMIN);
		//u3.setFirstName("Federico");
		//u3.setSurname("Sannasardo");
		//u3.setUsername("fedex");
		//u3.setPassword(Encryption.encrypt("fedex"));
		//u3.setPrivileges(Privileges.USER);
		//UserService.uDao.createUser(u, DBconnection.connecToDB());
		//UserService.uDao.createUser(u2, DBconnection.connecToDB());
		//UserService.uDao.createUser(u3, DBconnection.connecToDB());
	}
}
